proxy/src/timeout.rs

```diff
@@ -153,8 +159,9 @@ where
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match *self {
             TimeoutError::Timeout(ref duration) =>
-                // TODO: format the duration nicer.
-                write!(f, "operation timed out after {:?}", duration),
+                write!(f, "operation timed out after {}",
+                    HumanDuration(*duration)
+                ),
```

Comment:
also, i'd rather let rustfmt sort all of this out, but i prefer wrapping multiline match blocks with`{}` (even when there's a single statement).

Also, I think we should standardize at 100char line breaks, which I think this means we could collapse this onto a single line (especially if we rename `duration` to `d`, which seems fine in this context).

Reply:
Okay, I prefer 80-character lines but can accept 100! :)